xscen/catalog.py

```diff
@@ -347,6 +347,115 @@ def exists_in_cat(self, **columns):
             logger.info(f"An entry exists for: {columns}")
         return exists
 
+    def to_dataset(
+        self,
+        concat_on: Optional[Union[List[str], str]] = None,
+        ensemble_on: Optional[Union[List[str], str]] = None,
+        calendar: Optional[str] = "standard",
+        **kwargs,
+    ) -> xr.Dataset:
+        """
+        Open the catalog's entries into a single dataset.
+
+        Same as :py:meth:`~intake_esm.core.esm_datastore.to_dask`, but with additionnal control over the aggregations.
+        Ensemble preprocessing logic is taken from :py:func:`xclim.ensembles.create_ensemble`.
+        When `ensemble_on` is given, the function ensures all entries have the correct time coordinate according to `xrfreq`.
+        If either argument is given, the "id" is reconstructed by removing mentions of aggregated columns.
+        This will override any "custom" id, ones not unstackable with :py:func:`~xscen.catalog.unstack_id`.
+
+        Parameters
+        ----------
+        concat_on : list of strings or str, optional
```

Comment:
```suggestion
        extra_concat_on : list of strings or str, optional
```
My issue here is that people might try to say `concat_on: 'time'` or something like that. It should be more obvious in the name that this is in addition to 'time'. I'm not a fan of `extra`, just using it as an example.

Reply:
Hum. I understand, but I felt the concatenation over time was so self-evident that this misunderstanding (passing `concat_on=['time']`) wouldn't really be an issue. I think your additions to the docstring should be enough?